### ISSUE TITLE
Jakarta quick-fix fails due to getData() is null

### ISSUE THREAD

**Issue by turkeylurkey** [OpenLiberty/liberty-tools-vscode#336]
Certain quick fixes related to Jakarta do not appear with the diagnostic in the editor because of an exception. 

You can see the exception in the Output tab when you select "Language Support for Java." 
```
[Error - 12:57:13 PM] Request textDocument/codeAction failed.
  Message: Request jakarta/java/codeaction failed with message: java.lang.NullPointerException: Cannot invoke "Object.toString()" because the return value of "org.eclipse.lsp4j.Diagnostic.getData()" is null
  Code: -32603
```

An example is the quick fix reading "The `@AssertTrue` annotation can only be used on boolean and Boolean type fields."

More details are available [in the lsp4jakarta repository.](https://github.com/eclipse/lsp4jakarta/issues/416)

**Comment by aparnamichael** [comment]
Raised a PR with fix in lsp4Jakarta repo - https://github.com/eclipse/lsp4jakarta/pull/536

**Comment by TrevCraw** [comment]
@aparnamichael now that https://github.com/eclipse/lsp4jakarta/pull/536 has been merged, please verify this fix is resolved using the latest LSP4Jakarta 0.2.2-SNAPSHOT build. You will need to manually delete the language server jars and then re-run `npm run build`. Make sure LSP4Jakarta 0.2.2-SNAPSHOT is specified in all the correct locations. You can refer to https://github.com/OpenLiberty/liberty-tools-vscode/issues/361.

**Comment by aparnamichael** [comment]
Tested with LSP4Jakarta 0.2.2-SNAPSHOT, the null pointer exception is resolved, and the quick fixes are functioning as expected.
